docs/user_guide.md

```diff
@@ -163,6 +163,36 @@ Content-Length: 537
 }
 ```
 
+#### Incoming Webhook
+
+We can execute job with incoming webhook. Incoming webhook uses basic authorization. We can set key-value on `api_basic_authentication_applications` in kuroko2.yml.
```

Comment:
I think `basic authentication` is correct rather than `basic authorization`.
How about writing like below?

Jobs can be executed by incoming webhook.
Incoming webhook uses basic authentication and authentication information can be set at `api_basic_authentication_applications` key in kuroko2.yml.